cpp/dolfinx/mesh/MeshTags.h

```diff
@@ -76,6 +76,32 @@ class MeshTags
   /// Move assignment
   MeshTags& operator=(MeshTags&& tags) = default;
 
+  /// Find all MeshTags for a list of values and return the corresponding
+  /// indices per value, local-to-process
+  /// @param[in] values A list values
+  /// @return The corresponding indices
+  const std::vector<std::vector<std::int32_t>>&
+  indices(std::vector<T> values) const
```

Comment:
@garth-wells surely meant to put `const std::vector<T>& values` as an argument, not to return const vector by reference. What you have here is wrong.